src/components/native/ImageBackground.tsx

```diff
@@ -24,34 +24,42 @@ const UnistylesImageBackground = forwardRef<unknown, Props>((props, forwardedRef
             {...props}
             style={styleClassNames}
             imageStyle={imageClassNames}
-            ref={isServer() ? undefined : ref => {
-                if (!ref) {
-                    // @ts-expect-error hidden from TS
-                    UnistylesShadowRegistry.remove(storedRef, styleClassNames?.hash)
-                }
-
-                storedRef = ref
-                // @ts-expect-error hidden from TS
-                UnistylesShadowRegistry.add(ref, styleClassNames?.hash)
-
-                if (typeof forwardedRef === 'function') {
-                    return forwardedRef(ref)
-                }
-
-                if (forwardedRef) {
-                    forwardedRef.current = ref
-                }
-            }}
-            imageRef={isServer() ? undefined : ref => {
-                if (!ref) {
-                    // @ts-expect-error hidden from TS
-                    UnistylesShadowRegistry.remove(storedImageRef, imageClassNames?.hash)
-                }
-
-                storedImageRef = ref
-                // @ts-expect-error hidden from TS
-                UnistylesShadowRegistry.add(ref, imageClassNames?.hash)
-            }}
+            ref={
+                isServer()
```

Comment:
what about it ? Can we tweak it? Looks bad

Reply:
Same as comment below, related to ternary formatting https://github.com/biomejs/biome/discussions/2388